client-config/src/main/java/com/palantir/conjure/java/client/config/ClientConfigurations.java

```diff
@@ -73,11 +72,10 @@ public static ClientConfiguration of(ServiceConfiguration config) {
                 .readTimeout(config.readTimeout().orElse(DEFAULT_READ_TIMEOUT))
                 .writeTimeout(config.writeTimeout().orElse(DEFAULT_WRITE_TIMEOUT))
                 .enableGcmCipherSuites(config.enableGcmCipherSuites().orElse(DEFAULT_ENABLE_GCM_CIPHERS))
-                .fallbackToCommonNameVerification(config.fallbackToCommonNameVerification()
-                        .orElse(DEFAULT_FALLBACK_TO_COMMON_NAME_VERIFICATION))
-                .proxy(config.proxy()
-                        .map(ClientConfigurations::createProxySelector)
-                        .orElseGet(ProxySelector::getDefault))
+                .fallbackToCommonNameVerification(
+                        config.fallbackToCommonNameVerification().orElse(DEFAULT_FALLBACK_TO_COMMON_NAME_VERIFICATION))
+                .proxy(config.proxy().map(ClientConfigurations::createProxySelector).orElseGet(ProxySelector
+                        ::getDefault))
```

Comment:
so.. this is the kind of sadness enabled by the quick hack in https://github.com/palantir/palantir-java-format/pull/44